### PR TITLE
[stable/prometheus-blackbox-exporter] Add serviceAccount support

### PR DESCRIPTION
**Pull request by uncycler**
Signed-off-by: Frederic Marchand <fred@mautadine.com>

#### What this PR does / why we need it:
It adds serviceAccount to deployment

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/prometheus-blackbox-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus Blackbox Exporter
 name: prometheus-blackbox-exporter
-version: 3.2.0
+version: 3.2.1
```
Comment (desaintmartin) [inline stable/prometheus-blackbox-exporter/Chart.yaml]: Since this is a new feature, could you bump a MINOR version instead of PATCH?

**Comment by desaintmartin** [comment]
Thanks!
/assign
/ok-to-test

**Review by desaintmartin — CHANGES_REQUESTED** [review]

File: stable/prometheus-blackbox-exporter/Chart.yaml (hunk 1 above)
Comment (uncycler) [reply stable/prometheus-blackbox-exporter/Chart.yaml]: done

**Pull request merged**